encord/dataset.py

```diff
@@ -260,7 +260,7 @@ def create_image_group(
 
     def create_dicom_series(
         self,
-        file_paths: List[str],
+        file_paths: Union[List[str], List[Path], List[Union[Path, str]]],
```

Comment:
`s/List/Iterator/g` if we're going for the most lenient definition for users convenience, like in the other method. Otherwise just `Iterator[Path]` instead of the whole union in both.